backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Controllers/LessonsController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using EasySpeak.Core.BLL.Interfaces;
+using EasySpeak.Core.Common.DTO.Lesson;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EasySpeak.Core.WebAPI.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class LessonsController : ControllerBase
+    {
+        private readonly ILessonsService _lessonsService;
```

Comment:
New line after list of private members + check other controllers and where it's localted

Reply:
fixed